tias/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```

Comment:
We may want to disable some crawlers, particularly if we don't lock down our pages very tightly.